tests/rbd/rbd_utils.py

```diff
@@ -204,6 +204,19 @@ def check_pool_exists(self, pool_name: str) -> bool:
             )
             return False
 
+    def export(self, imagespec, path):
```

Comment:
This method is already added in rbd_uitls in one of the latest PR merge https://github.com/red-hat-storage/cephci/pull/2393 , Please rebase your branch.

Reply:
Addressed, thank you